packages/core/src/node/process/process.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, decorate } from 'inversify';
+import { ILogger } from '../../common/logger';
+import * as events from 'events';
+import * as child from 'child_process';
+import * as stream from 'stream';
+
+decorate(injectable(), events.EventEmitter);
+@injectable()
+export abstract class Process extends events.EventEmitter {
```

Comment:
Yes I was unsure about that so I discussed it with @akosyakov on Gitter and he said using node's event emitter was fine for node code and that our own event emitter should be used for front end code...

Also, I did a implementation using both and I think @akosyakov is right since we often don't have the choice to deal node's event emitters anyway and it kinda simplifies the code especially when forwarding events from another node event emitter like I do.

But I'm open to discussion, we should make it a formal policy to avoid this kind of problem in the future.

WDTY?